modules/rdma-creating-persistent-naming-rules.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/rdma-remote-direct-memory-access.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rdma-creating-persistent-naming-rules_{context}"]
+
+= Creating persistent naming rules
+
+In some cases, device names won't persist following a reboot. For example, on R760xa systems 
+Mellanox devices might be renamed after a reboot. You can avoid this problem by using a `MachineConfig` to set persistence.
+
+.Procedure 
+
+. Gather the MAC address names from the worker nodes for the node into a file and provide names for the interfaces that need to persist. This example uses the file `70-persistent-net.rules` and stashes the details in it.
++
+[source,terminal]
+----
+$ cat <<EOF > 70-persistent-net.rules
+SUBSYSTEM=="net",ACTION=="add",ATTR{address}=="b8:3f:d2:3b:51:28",ATTR{type}=="1",NAME="ibs2f0"
+SUBSYSTEM=="net",ACTION=="add",ATTR{address}=="b8:3f:d2:3b:51:29",ATTR{type}=="1",NAME="ens8f0np0"
+SUBSYSTEM=="net",ACTION=="add",ATTR{address}=="b8:3f:d2:f0:36:d0",ATTR{type}=="1",NAME="ibs2f0"
+SUBSYSTEM=="net",ACTION=="add",ATTR{address}=="b8:3f:d2:f0:36:d1",ATTR{type}=="1",NAME="ens8f0np0"
+EOF
+----
+
+. Convert that file into a base64 string without line breaks and set the output to the variable `PERSIST`:
++
+[source,terminal]
+----
+$ PERSIST=`cat 70-persistent-net.rules| base64 -w 0`
+
+$ echo $PERSIST
+U1VCU1lTVEVNPT0ibmV0IixBQ1RJT049PSJhZGQiLEFUVFJ7YWRkcmVzc309PSJiODozZjpkMjozYjo1MToyOCIsQVRUUnt0eXBlfT09IjEiLE5BTUU9ImliczJmMCIKU1VCU1lTVEVNPT0ibmV0IixBQ1RJT049PSJhZGQiLEFUVFJ7YWRkcmVzc309PSJiODozZjpkMjozYjo1MToyOSIsQVRUUnt0eXBlfT09IjEiLE5BTUU9ImVuczhmMG5wMCIKU1VCU1lTVEVNPT0ibmV0IixBQ1RJT049PSJhZGQiLEFUVFJ7YWRkcmVzc309PSJiODozZjpkMjpmMDozNjpkMCIsQVRUUnt0eXBlfT09IjEiLE5BTUU9ImliczJmMCIKU1VCU1lTVEVNPT0ibmV0IixBQ1RJT049PSJhZGQiLEFUVFJ7YWRkcmVzc309PSJiODozZjpkMjpmMDozNjpkMSIsQVRUUnt0eXBlfT09IjEiLE5BTUU9ImVuczhmMG5wMCIK
+----
+
+. Create a machine configuration and set the base64 encoding in the custom resource file:
++
+[source,terminal]
+----
+$ cat <<EOF > 99-machine-config-udev-network.yaml
+----
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+   labels:
+     machineconfiguration.openshift.io/role: worker
+   name: 99-machine-config-udev-network
+spec:
+   config:
+     ignition:
+       version: 3.2.0
+     storage:
+       files:
+       - contents:
+           source: data:text/plain;base64,$PERSIST
+         filesystem: root
+         mode: 420
+         path: /etc/udev/rules.d/70-persistent-net.rules
```

Comment:
Just checking - should these two blocks be combined if the command is seemingly supposed to add content to a file for everything in between the command and `EOF`? Maybe I don't fully understand, disregard if that's the case
```suggestion
$ cat <<EOF > 99-machine-config-udev-network.yaml
apiVersion: machineconfiguration.openshift.io/v1
kind: MachineConfig
metadata:
   labels:
     machineconfiguration.openshift.io/role: worker
   name: 99-machine-config-udev-network
spec:
   config:
     ignition:
       version: 3.2.0
     storage:
       files:
       - contents:
           source: data:text/plain;base64,$PERSIST
         filesystem: root
         mode: 420
         path: /etc/udev/rules.d/70-persistent-net.rules
EOF
```